backend/src/test/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceIntegrationTest.java

```diff
@@ -0,0 +1,196 @@
+package gov.cdc.usds.simplereport.service;
+
+import static gov.cdc.usds.simplereport.api.Translators.parsePhoneType;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.api.uploads.PatientBulkUploadResponse;
+import gov.cdc.usds.simplereport.db.model.Facility;
+import gov.cdc.usds.simplereport.db.model.Person;
+import gov.cdc.usds.simplereport.db.model.PhoneNumber;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonRole;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PhoneType;
+import gov.cdc.usds.simplereport.db.model.auxiliary.StreetAddress;
+import gov.cdc.usds.simplereport.db.model.auxiliary.UploadStatus;
+import gov.cdc.usds.simplereport.db.repository.PhoneNumberRepository;
+import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration.Role;
+import gov.cdc.usds.simplereport.test_util.TestUserIdentities;
+import java.io.InputStream;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = "hibernate.query.interceptor.error-level=ERROR")
+@WithMockUser(
+    username = TestUserIdentities.SITE_ADMIN_USER,
+    authorities = {Role.SITE_ADMIN, Role.DEFAULT_ORG_ADMIN})
+class PatientBulkUploadServiceIntegrationTest extends BaseServiceTest<PatientBulkUploadService> {
```

Comment:
I just renamed the existing test, there's nothing actually different about the setup/configuration.

I just got annoyed as I was importing several dependency services and figured we should call a spade a spade 🙃